### ISSUE TITLE
Valid clabe reported as invalid

### ISSUE THREAD

**Issue by a0nguyen** [center-key/clabe-validator#21]
A valid key is reported as invalid

## Expected Behavior
012660015490725198 is a valid key from BBVA and I expect it to be marked as valid
![Imagen de iOS](https://user-images.githubusercontent.com/6928214/87100519-373a7800-c212-11ea-949a-a58e9afe4437.jpg)
Also, it is marked as valid using this tool --> https://bank.codes/mexico-clabe-checker/ (but I love center-key/clabe-validator so I wont use the other)

## Current Behavior
012660015490725198 is marked as invalid --> Invalid city

## Steps to Reproduce
https://jsfiddle.net/Lvnqdr0g/

**Comment by pilafmon** [comment]
Release v1.3.8

```shell
% node
Welcome to Node.js v14.5.0.
Type ".help" for more information.
> const clabe = require('./clabe');
undefined

> clabe.validate('012660015490725198');
{
  ok: true,
  error: null,
  formatOk: true,
  message: 'Valid',
  tag: 'BBVA BANCOMER',
  bank: 'BBVA Bancomer, S.A.',
  city: 'Huejotzingo',
  account: '01549072519',
  code: { bank: '012', city: '660' },
  checksum: 8
}
```
